### ISSUE TITLE
string(74) "(#803) Some of the aliases you requested do not exist:

### ISSUE THREAD

**Issue by kuldeep1984** [SammyK/LaravelFacebookSdk#28]
Hi,

I am trying to post to a specific page with the following code but this return the error.

 Facebook::setAccessToken('####################');
        $page_id = ####################;
        $data = array(
            'link' => 'http://google.com/',
            'message' => 'primer on geospatial data and mongodb', 
        );

        try {
            $facebook_event = Facebook::object('/' . $page_id . '/feed')->with($data)->post();
        } catch (\SammyK\FacebookQueryBuilder\FacebookQueryBuilderException $e) {
            dd($e->getPrevious()->getMessage());
        }

the Error is -
string(74) "(#803) Some of the aliases you requested do not exist:

**Comment by kuldeep1984** [comment]
Well... I find out the error! The Page access token was expired!

So the question is how to get long lived access token for page

**Comment by SammyK** [comment]
Hey @kuldeep1984! Glad you found the solution to your problem!

I wrote up a [guide about access tokens](https://www.sammyk.me/access-token-handling-best-practices-in-facebook-php-sdk-v4) that might help you.

**TL;DR:** Use a long-lived user access token to obtain the page access token and the page access token will never expire.

Good luck!
